### PR TITLE
Add support for linter configuration files

### PR DESCRIPTION
**Pull request by balloob**
This pull request adds support for configuration files for your linters.

As discussed last week, Farcy will look for configuration files in the folder `~/.config/farcy/handler_{name}.conf`.

### CODE REVIEW

File: farcy/handlers.py [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 from __future__ import print_function
 from collections import defaultdict
+from .const import CONFIG_DIR
```
Comment (bboe) [inline farcy/handlers.py]: Can you move this to just above `.exceptions`? I think the general practice is to do system imports prior to local imports.

**Comment by balloob** [comment]
Am I correct in assuming that you prefer to order partial imports above full imports, system above local?

What about third party imports?

**Comment by bboe** [comment]
I was just looking at some other projects and it turns out they differ from what I was thinking. I guess as long as it's consistent across the module I'm okay with whatever.

**Comment by balloob** [comment]
The imports have been updated. Good to go?

**Comment by bboe** [comment]
:+1:

**Pull request merged**